<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>598</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>598</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QPushButton" name="RecordPart1_Btn">
    <property name="geometry">
     <rect>
      <x>412</x>
      <y>370</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 84, 115);
background-image: url();
border-radius: 15px;
font-size: 12px;
font-weight:bold;
color: rgb(255, 252, 255)
</string>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QLabel" name="RecordPart1_Bar_Label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>318</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
color:rgb(255, 252, 255);
font-size: 12px;</string>
    </property>
    <property name="text">
     <string>0:00/0:30</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="RecordPart1_Slider">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>317</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{background-image: url();}
QSlider::groove:horizontal { 
	background-color: rgb(255, 255, 0);
	border: 0px solid #424242; 
	height: 4px; 
	border-radius: 6px;
}
QSlider::handle:horizontal { 
	background-color:rgb(62, 61, 163); 
	border: 2px solid rgb(62, 61, 163);  
	width: 8px; 
	height: 10px; 
	line-height: 20px; 
	margin-top: -5px; 
	margin-bottom: -5px; 
	border-radius: 6px; 
}

QSlider::handle:horizontal:hover { 
	border-radius: 6px; 
}
QSlider::add-page:horizontal {
    background: white;
}

QSlider::sub-page:horizontal {
    background: rgb(255, 84, 115);
}</string>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>576</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RecordPart2_Bar_Label">
    <property name="geometry">
     <rect>
      <x>755</x>
      <y>318</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();
color:rgb(255, 252, 255);
font-size: 12px;</string>
    </property>
    <property name="text">
     <string>0:00/0:30</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSlider" name="RecordPart2_Slider">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>318</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{background-image: url();}
QSlider::groove:horizontal { 
	background-color: rgb(255, 255, 0);
	border: 0px solid #424242; 
	height: 4px; 
	border-radius: 6px;
}
QSlider::handle:horizontal { 
	background-color:rgb(62, 61, 163); 
	border: 2px solid rgb(62, 61, 163);  
	width: 8px; 
	height: 10px; 
	line-height: 20px; 
	margin-top: -5px; 
	margin-bottom: -5px; 
	border-radius: 6px; 
}

QSlider::handle:horizontal:hover { 
	border-radius: 6px; 
}
QSlider::add-page:horizontal {
    background: white;
}

QSlider::sub-page:horizontal {
    background: rgb(255, 84, 115);
}</string>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="RecordPart2_Btn">
    <property name="geometry">
     <rect>
      <x>767</x>
      <y>370</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 84, 115);
background-image: url();
border-radius: 15px;
font-size: 12px;
font-weight:bold;
color: rgb(255, 252, 255)
</string>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EndSession_btn">
    <property name="geometry">
     <rect>
      <x>825</x>
      <y>445</y>
      <width>160</width>
      <height>66</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 84, 115);
background-image: url();
border-radius: 20px;
font-size: 18px;
font-weight:bold;
color: rgb(255, 252, 255)</string>
    </property>
    <property name="text">
     <string>End Session</string>
    </property>
   </widget>
   <widget class="QLabel" name="RecordPart1_ErrorMsg">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0);
font-size: 14px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="RecordPart2_ErrorMsg">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0);
font-size: 14px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>RecordPart1_Btn</zorder>
   <zorder>RecordPart1_Bar_Label</zorder>
   <zorder>RecordPart1_Slider</zorder>
   <zorder>RecordPart2_Bar_Label</zorder>
   <zorder>RecordPart2_Slider</zorder>
   <zorder>RecordPart2_Btn</zorder>
   <zorder>EndSession_btn</zorder>
   <zorder>RecordPart1_ErrorMsg</zorder>
   <zorder>RecordPart2_ErrorMsg</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
